src/views/enrollUser/Footer.js

```diff
@@ -0,0 +1,33 @@
+/*!
```

Comment:
will it make sense to add this file to `views/shared` as a `FooterBackLink.js` or similar? Seems it is generic enough that it can be reused